src/common/UnreadCount.js

```diff
@@ -71,7 +71,7 @@ export default class UnreadCount extends PureComponent<Props> {
 
     return (
       <View style={frameStyle}>
-        <Text style={textStyle} numberOfLines={1}>
+        <Text style={textStyle} numberOfLines={1} ellipsizeMode="clip">
```

Comment:
@borisyankov  The value **clip** for **ellipsizeMode**  prop only works on ios and not on Android. I am getting the following error while running the app on my simulator(Nexus 4 API 23 Android 6.0).
![rcttext](https://user-images.githubusercontent.com/24224542/37595985-aaf13cb2-2ba0-11e8-9f81-f840324b177c.png)